matrix-sdk/src/main/java/org/matrix/androidsdk/data/Room.java

```diff
@@ -2385,14 +2387,34 @@ public void onUnexpectedError(Exception e) {
         if (isEncrypted() && (null != mDataHandler.getCrypto())) {
             mDataHandler.updateEventState(event, Event.SentState.ENCRYPTING);
 
+            // Store the "m.relates_to" data and remove them from event content before encrypting the event content
+            final JsonElement relatesTo;
+
+            if (event.getContent().isJsonObject()
+                    && event.getContent().getAsJsonObject().has("m.relates_to")) {
```

Comment:
Seems like there is a getContentAsJsonObject method

Reply:
ok, fixed